src/Common/Core/Test/Microsoft.Common.Core.Test.csproj

```diff
@@ -44,7 +44,7 @@
       <Project>{01e3d8c2-9c24-492b-b13b-c6d7068b6bf8}</Project>
       <Name>Microsoft.R.Common.Core.Windows</Name>
     </ProjectReference>
-    <ProjectReference Include="..\Impl\Microsoft.Common.Core.csproj">
+    <ProjectReference Include="..\Impl\Microsoft.R.Common.Core.csproj">
```

Comment:
Maybe call it `Microsoft.Rtvs.Common.Core.csproj`, cause we use `.R.` for projects that have code related to R language.